src/scripts/main.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    const handler = () => {
+      element.removeEventListener(eventName, handler);
+
+      resolve(`It was ${eventName}
+        on the element: ${element.nodeName}, id: ${element.id}.`);
```

Comment:
There is a syntax error here due to a missing closing quote in the string. Ensure that the string is correctly enclosed within quotes to avoid runtime errors.